flocker/provision/_install.py

```diff
@@ -43,6 +43,103 @@
 }
 
 
+def task_client_smoke_test():
+    """
+    Check that the CLI is working.
+    """
+    return run_from_args(['flocker-deploy', '--version'])
+
+
+def task_install_cli(distribution, package_source=PackageSource()):
+    """
+    Install flocker CLI on a distribution.
```

Comment:
Put a brief explanation of the differences which occur if `package_source.branch` or not.